docs/development/install.rst

```diff
@@ -1,194 +1,267 @@
-Installation
-============
+Install local development instance
```

Comment:
Think we can just call this installation, so is easier to find when searching.

Reply:
Maybe `Development Installation` or similar?